Engine/Component/ComponentBillboard.h

```diff
@@ -31,9 +31,7 @@ class ComponentBillboard : public Component
 
 	ComponentBillboard();
 	ComponentBillboard(GameObject* owner);
-	~ComponentBillboard();
-
-	void CleanUp();
+	~ComponentBillboard() = default;
```

Comment:
you need to delete the quad on the destructor

Reply:
is not needed as I'm not allocating memory for the quad, it is managed on the module